cartography/intel/aws/ec2/load_balancer_v2s.py

```diff
@@ -157,8 +157,10 @@ def load_load_balancer_v2_target_groups(
     ingest_instances = """
     MATCH (elbv2:LoadBalancerV2{id: $ID}), (instance:EC2Instance{instanceid: $INSTANCE_ID})
     MERGE (elbv2)-[r:EXPOSE]->(instance)
-    ON CREATE SET r.firstseen = timestamp()
-    SET r.lastupdated = $update_tag
+    ON CREATE SET r.firstseen = timestamp(),
+                  r.target_group_arn = $TARGET_GROUP_ARN
```

Comment:
Is it possible for the target_group_arn to change? To be safe, let's move this to the SET clause.